leptonai/templates/stable_diffusion_webui/README.md

```diff
@@ -0,0 +1,73 @@
+# Stable Diffusion Web UI
+
+Stable Diffusion Web UI is a web browser interface tool to easily manage model checkpoints, loras, and configurations for image generation tasks with the Stable Diffusion model, an AI model can generate images from text prompts or modify existing images with text prompts. This documentation will walk you through setting up your environment, adding your choice of models or Loras, and generating an image.
+
+# Configurations
+
+Here are the configurations you can set for your deployment:
```

Comment:
So the configuration is going to be a bit tricky. I think:
- name is fine.
- resource shape is fine
- mount is going to be tricky, since right now I don't think the template page parses / supports mount. cc'ing @vthinkxie to take a look.

Reply:
track https://github.com/leptonai/lepton/issues/5294